File: src/java/fizzbuzz.java
public class fizzbuzz {

    public static void main(String[] args) {
        for (int i = 1; i <= 100; i++){
            if (i % 3 == 0 && i % 5 == 0)
                System.out.print("FizzBuzz");
            else if (i % 3 == 0)
                System.out.print("Fizz");
            else if (i % 5 == 0)
                System.out.print("Buzz");
            else
                System.out.print(i);
            
            if (i % 10 == 0)
                System.out.println();
            else
                System.out.print(",");
        }
    }

}

File: src/c/fizzbuzz.c
#include <stdio.h>

int main()
{
    int i;
    for (i = 1; i <= 100; i++)
    {
        if (i % 3 == 0 && i % 5 == 0)
            printf("FizzBuzz");
        else if (i % 3 == 0)
            printf("Fizz");
        else if (i % 5 == 0)
            printf("Buzz");
        else
            printf("%i", i);

        if (i % 10 == 0)
            printf("\n");
        else
            printf(",");
    }
    
    return 0;
}